README.md

```diff
@@ -35,6 +35,33 @@ Commands should be run from inside the theme folder: `web/app/themes/*`.
 - For watching and buildint, run `yarn watch` or `yarn build`
 - For building production assets, run `yarn build:production`
 
+### Run with Docker
+
+Beside using a local webserver you can use Docker Compose to quickly start a webserver with a database to run your WordPress Scaffold project.
+
+```bash
+# Create your personal configuration by coping the opinionated default configuration
+cp docker-compose.development.yml docker-compose.override.yml
+```
+
+The default port for Apache is 40010. To change it replace 40010 at `services.front-end.ports` and change `services.php.environment.WP_HOME` accordingly.
+
+The default port for MySQL is 40011. To change it replace 40011 at `services.db.ports`.
+
+On MacOS Docker is quite slow. That's due to I/O performance issues. To bypass those problems NFS could be used. Uncomment the lines in `docker-compose.override.yml` to create a NFS volume to access the application code. To configure NFS [read this tutorial on Medium](https://medium.com/@sean.handley/how-to-set-up-docker-for-mac-with-native-nfs-145151458adc).
+
+```bash
+# Start Docker Compose
+docker-compose up
+
+# Open the website
+open http://localhost:40010
+
+# Connect to wordpress database from host with root user
+mysql -h 127.0.0.1 -P 40011 -u root -p wordpress
+# The password can be found in `docker-compose.yml`
```

Comment:
Wouldn't it be better to promote the use of `docker-compose exec db mysql -uroot etc`?
Not everyone might have a `mysql` client installed on their system.

Reply:
Dat is een goede. Scheelt weer een dependency.